metadata-ingestion/src/datahub/ingestion/source/sql/redshift.py

```diff
@@ -170,10 +169,6 @@ class RedshiftConfig(
         description="Which table lineage collector mode to use. Available modes are: [stl_scan_based, sql_based, mixed]",
     )
 
-    @pydantic.validator("platform")
```

Comment:
what is this? we are just removing platform from bigquery completely?

Reply:
Yup - if the `platform` needs to always be "bigquery", why is it even a config? Also it turned out that we weren't really ever using that config anyways